src/app/badges/badges.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-badges',
+  templateUrl: './badges.component.html',
+  styleUrls: ['./badges.component.scss']
+})
+export class BadgesComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
```

Comment:
Please remove empty constructor and ngOnInit

Reply:
fixed